### PR TITLE
Assets & Readings: Added refresh icon on Asset summary and Asset readings modal

### PR DESCRIPTION
**Pull request by monika-Dianomic**

### CODE REVIEW

File: src/app/components/core/asset-readings/asset-summary/asset-summary.component.html [hunk 1]
```diff
@@ -2,7 +2,12 @@
   <div class="modal-background"></div>
   <div class="modal-card">
     <header class="modal-card-head">
-      <p [hidden]='!isValidData' class="modal-card-title is-size-5">{{assetCode}}</p>
+      <p [hidden]='!isValidData' class="modal-card-title is-size-5">{{assetCode}}
```
Comment (praveen-garg) [inline src/app/components/core/asset-readings/asset-summary/asset-summary.component.html]: isValidData ?

File: src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts [hunk 2]
```diff
@@ -40,13 +41,12 @@ export class ReadingsGraphComponent implements OnInit {
...
-    this.isValidData = true;
     this.isReadingsAvailable = true;
     this.isInvalidLimit = false;
     this.isInvalidOffset = false;
-
-    if (limit === undefined || limit === '') {
-      limit = 0;
+    this.showGraph = true;
+    if (limit === undefined || limit === '' || limit === 0) {
+      limit = 100;
```
Comment (praveen-garg) [inline src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts]: how it was set to 20 before?

File: src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts [hunk 3]
```diff
@@ -85,12 +92,15 @@ export class ReadingsGraphComponent implements OnInit {
                 console.log('readings data to show trends.', this.assetReadingSummary);
               });
           } else {
-            this.isValidData = false;
+            this.showGraph = false;
+            const labels = [];
+            const ds = [];
+            this.setAssetReadingValues(labels, ds);
```
Comment (praveen-garg) [inline src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts]: Please clarify the above, w.r.t log in next line

File: src/app/components/core/asset-readings/asset-summary/asset-summary.component.html (hunk 1 above)
Comment (praveen-garg) [reply src/app/components/core/asset-readings/asset-summary/asset-summary.component.html]: @monika-Dianomic @mshariq-nerd  still needs to be resolved!

File: src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts [hunk 4]
```diff
@@ -199,7 +203,7 @@ export class ReadingsGraphComponent implements OnInit {
         backgroundColor: '#239B56',
         borderColor: '#82E0AA',
       }];
-    } else {
+    } else if (assetReading.length === 1) {
```
Comment (praveen-garg) [inline src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts]: what is the case here? when length ==1

Comment (mshariq-nerd) [reply src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts]: isValidData is not changed in asset-summary.component. Working as it is.

that length check is to check if there is any data in there into dataset.

**Pull request merged**